usdf/batch.rst

```diff
@@ -4,12 +4,23 @@ Batch Resources
 
 This document describes the batch resources available at the US Data Facility hosted at the SLAC Shared Scientific Data Facility (S3DF).
 
-Use of S3DF batch is documented at 
+Use of S3DF batch is documented at
 
 https://s3df.slac.stanford.edu/
 
+and more general Slurm documentation is available at https://slurm.schedmd.com/.
+
 S3DF has two Slurm partitions ``roma`` and ``milano``. Both of these partitions have AMD nodes with 128 cores, composed of two sockets, each with a 64-core processor (hyperthreading disabled).
-A Slurm job can be submitted with markup ``#SBATCH -p roma,milano`` to run on either of the partitions. 
+A Slurm job can be submitted with markup ``#SBATCH -p roma,milano`` to run on either of the partitions.
+
+For light interactive work, e.g., running *small* ``pipetask`` jobs, one can obtain an interactive session on the batch nodes using ``srun``.  For example,
+
+.. code-block:: bash
+   :name: srun-interactive-example
+
+   srun --pty --cpus-per-task=4 --mem=6GB --time=02:00:00 --partition=roma,milano --account=rubin bash
+
+will create a 2-hour, 4-core bash session with a total of 6GB memory.  One can then run ``pipetask -j 4``, making use of the 4 allocated cores.  In general, using BPS is preferred to running ``pipetask`` directly since many concurrent ``pipetask`` jobs that are run like this can cause registry database contention.
```

Comment:
I think we're supposed to add `--nodes=1` to make sure that all our cpus are put on one node (but I don't know what `--pty` does otherwise).  And maybe change the `6GB` to something like `16GB`, the "fair share" for 4 CPUs.  Also, I think it's worth mentioning that specifying the total memory rather than per core is probably what the user wants because otherwise if any process goes over the per-core memory threshold the whole thing gets killed.
And it is also worth mentioning `--exclusive` which will just give you a whole node if that's what you need.

Reply:
Thanks, @erykoff .   I've made some changes following your suggestions.  If you give me thumbs up on those, I'll go ahead and merge.

Reply:
Looks fine.  But note that sometimes I just want a whole node to run fgcmcal in debugging mode on a large dataset.  😄